serde/byte.go

```diff
@@ -0,0 +1,38 @@
+package serde
```

Comment:
Does this have anything to do with https://github.com/serde-rs/serde ?

Reply:
Nothing besides a cool name that succinctly combines deserialization and serialization in one word 😄